resources/NBTest/NBTest_062_TestRouteHandler.ipynb

```diff
@@ -10,7 +10,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "imported m, np, pd, plt, os, sys, decimal; defined iseq, raises, require\n",
+      "imported m, np, pd, plt, os, sys, decimal; defined iseq, raises, require, Timer\n",
```

Comment:
How is this change related to the problem that this PR is attempting to fix?

Reply:
I did not manually change this specifically. I added a test to this Notebook to test the function that I added - native_gas_token_to_wrapped

Reply:
Well, where is that test then?
Something doesn't make sense here.

Reply:
NBTest_062_TestRouteHandler.py